blocksuite/affine/data-view/src/view-presets/table/table-view-selector.ts

```
@@ -0,0 +1,45 @@
+export class TableViewSelector extends DataViewBase<
+  TableSingleView,
+  TableViewSelectionWithType
+> {
+  tableRef$ = signal<VirtualTableView | DataViewTable>();
+
+  // @ts-expect-error TODO: fix this
+  get expose() {
+    if (this.tableRef$.value) {
+      return this.tableRef$.value.expose;
+    }
+  }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)